src/e2e-test/java/io/cdap/plugin/BQValidation.java

```diff
@@ -223,6 +223,9 @@ public static boolean compareSalesforceAndJsonData(List<JsonObject> salesforceDa
             break;
 
           default:
+            if (columnName.equals("Col_GeoLocation__c")) {
```

Comment:
Done

Reply:
Please update the comment to specify *why* we are ignoring this column in the test. i.e. what would happen if we didn't ignore this column?

Reply:
We are ignoring this column because this is auto-generated field in salesforce and it was combining the values for latitude and longitude in this column and ideally it should go in double but it is going in string and the their data types are in the form jsonprimitive and jsonobject and the result I got was {"latitude":37.794016,"longitude":-122.395016}  and in bigquery it is {Col_GeoLocation__Longitude__s=-122.395016, Col_GeoLocation__Latitude__s=37.794016}".So that's why iIam ignoring that.

Reply:
Thanks. Please add this info as a comment in the code.

Reply:
done @35ca98bfef244596d0893b1148f028b2e25975be